src/server/handler.tsx

```diff
@@ -33,14 +33,14 @@ type DependencyMapItem = { modules: string[], path: string };
 
 type RenderHtmlType<T> =
   T extends true ? ReturnType<typeof renderToNodeStream> :
-    T extends false ? ReturnType<typeof renderToString> :
-      never;
+  T extends false ? ReturnType<typeof renderToString> :
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 4 spaces but found 2.](https://app.codacy.com/gh/Atyantik/pawjs/pullRequest?prid=8527366)
```suggestion
    T extends false ? ReturnType<typeof renderToString> :
```